docs/content/getting-started/configuration-overview.md

```diff
@@ -21,7 +21,7 @@ This configuration can change and is seamlessly hot-reloaded, without any reques
 
 ## The Dynamic Configuration
 
-Traefik gets its _dynamic configuration_ from [providers](../providers/overview.md): whether an orchestrator, a service registry, or a plain old configuration file.
+Traefik gets its _dynamic configuration_ from [providers](../providers/overview.md): whether an orchestrator, a service registry, or [additional configuration file](../providers/file.md) dedicated to _dynamic configuration_.
```

Comment:
```suggestion
Traefik gets its _dynamic configuration_ from [providers](../providers/overview.md): whether an orchestrator, a service registry, or a plain old configuration file.
```

This is not so much an additional configuration file, the file provider is looking for a file or files in a directory, but it is a provider on his own.

Reply:
I've updated this part according to your comment.